selftest/tests.py

```diff
@@ -65,7 +65,7 @@
 planpythontestsuite("none", "samba.tests.upgrade")
 planpythontestsuite("none", "samba.tests.core", py3_compatible=True)
 planpythontestsuite("none", "samba.tests.common")
-planpythontestsuite("none", "samba.tests.provision")
+planpythontestsuite("none", "samba.tests.provision", py3_compatible=True)
 planpythontestsuite("none", "samba.tests.password_quality")
 planpythontestsuite("none", "samba.tests.samba3")
```

Comment:
This line should be removed now.

Reply:
that's annoying, this must have been lost  during last rebase (was fixed previously)